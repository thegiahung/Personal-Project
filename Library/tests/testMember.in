member 100000
add member Python
add member Java
add member Ruby
member 100000
member 100001
member 100002
member 100003
member 100004
exit
