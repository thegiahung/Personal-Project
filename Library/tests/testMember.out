user: No members in system.

user: Success.

user: Success.

user: Success.

user: 100000: Python

user: 100001: Java

user: 100002: Ruby

user: No such member in system.

user: No such member in system.

user: Ending Library process.
